floodviz/static/js/map.js

```diff
@@ -61,10 +63,10 @@
 				.attr("transform", function (d) {
 					return "translate(" + projection(d.geometry.coordinates) + ")";
 				})
-				.attr("id", function(d) {
-					if (classname === 'gage-point') {
-						return 'map' + d.properties.id;
-					} else { return '';}
+				.attr("id", function (d) {
+					if (property_for_id && d.properties[property_for_id]) {
+						return 'map' + d.properties[property_for_id];
+					}
```

Comment:
Should be an else clause that returns on an empty string. Also consider using ternary operator, see https://www.w3schools.com/js/js_comparisons.asp rather than an if-else clause.